packages/ExpandCollapse/__tests__/ExpandCollapse.spec.jsx

```diff
@@ -249,30 +249,6 @@ describe('ExpandCollapse', () => {
     })
   })
 
-  describe('accessibility', () => {
-    it('marks the panel as an expandable trigger for content for screen readers', () => {
-      const { findPanelHeader, clickPanel } = doMount(
-        <ExpandCollapse>{aPanel({ id: 'panel-1' })}</ExpandCollapse>
-      )
-
-      expect(findPanelHeader('panel-1')).toHaveProp('aria-expanded', false)
-
-      clickPanel('panel-1')
-      expect(findPanelHeader('panel-1')).toHaveProp('aria-expanded', true)
-    })
-
```

Comment:
Why were the `aria-expanded` tests removed? Did you also remove `aria-expanded` from ExpandCollapse? If so, why? If not, please add this test back.